### PR TITLE
CASMCMS-8473: Run tftp file transger subtest on workers not masters

### PR DESCRIPTION
**Pull request by mharding-hpe**
main backport of https://github.com/Cray-HPE/csm/pull/1963

### CODE REVIEW

File: rpm/cray/csm/sle-15sp4/index.yaml [hunk 1]
```diff
@@ -29,13 +29,14 @@ https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp4/:
     - cfs-debugger-1.3.1-1.x86_64
     - cfs-state-reporter-1.9.1-1.x86_64
     - cfs-trust-1.6.3-1.x86_64
+    - cray-cmstools-crayctldeploy-1.11.1-1.x86_64
     - cray-site-init-1.31.0-1.x86_64
     - csm-node-identity-1.0.19-1.noarch
     - csm-ssh-keys-1.5.1-1.noarch
     - csm-ssh-keys-roles-1.5.1-1.noarch
-    - csm-testing-1.16.13-1.noarch
+    - csm-testing-1.16.14-1.noarch
```
Comment (mharding-hpe) [inline rpm/cray/csm/sle-15sp4/index.yaml:37]:
```suggestion
    - csm-testing-1.16.16-1.noarch
```

File: rpm/cray/csm/sle-15sp4/index.yaml [hunk 2]
```diff
@@ -29,13 +29,14 @@ https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp4/:
...
     - docs-csm-1.5.20-1.noarch
-    - goss-servers-1.16.13-1.noarch
+    - goss-servers-1.16.14-1.noarch
```
Comment (mharding-hpe) [inline rpm/cray/csm/sle-15sp4/index.yaml:39]:
```suggestion
    - goss-servers-1.16.16-1.noarch
```

**Comment by denniswalker** [comment]
Merged in with another PR for a newer version.

**Comment by mharding-hpe** [comment]
The other PR did not include all of the RPMs included in this PR. I'm reopening this one and resolving the merge conflicts.

**Pull request merged**